cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -73,13 +72,16 @@ const (
 	MostPodsExpanderName = "most-pods"
 	// LeastWasteExpanderName selects a node group that leaves the least fraction of CPU and Memory
 	LeastWasteExpanderName = "least-waste"
+	// Namespace in which cluster-autoscaler run
+	KubernetesNamespace = "kube-system"
```

Comment:
Running cluster autoscaler in kube-system namespace is recommended but not required.